setup.py

```diff
@@ -34,6 +34,14 @@
 
 
 def setup_package():
+    package_data = dict(
+        entry_points={
+            "console_scripts": [
+                "skops = skops.cli.entrypoint:main_entrypoint",
```

Comment:
Should we simply call it `entrypoint` instead of `main_entrypoint`?

Reply:
We can, I named it that way so it didn't clash with the file name

`entrypoint.entrypoint` feels a little clunky to me, happy for other suggestions or to go with that if anyone feels strongly

Reply:
I don't mind either way.

Reply:
maybe `parent_cli` or `main_cli`?